actors/evm/src/lib.rs

```diff
@@ -255,17 +270,33 @@ impl ActorCode for EvmContractActor {
                 let value = Self::storage_at(rt, cbor::deserialize_params(params)?)?;
                 Ok(RawBytes::serialize(value)?)
             }
-            None => Self::invoke_contract(rt, method, params),
+            None => Self::invoke_contract(
+                rt,
+                &EthAddress::from_id(rt.message().caller().id().unwrap()),
+                method,
+                params,
+            ),
         }
     }
 }
 
 #[derive(Serialize_tuple, Deserialize_tuple)]
 pub struct ConstructorParams {
-    pub bytecode: RawBytes,
+    /// The actor's "creator" (specified by the EAM).
+    pub creator: EthAddress,
```

Comment:
We'll need this regardless of where we store addresses.

Reply:
same comment applies here, are we forcing eth addresses everywhere?